library/core/src/main/java/com/google/android/exoplayer2/audio/AudioCapabilities.java

```diff
@@ -48,6 +48,9 @@
   /** Global settings key for devices that can specify external surround sound. */
   private static final String EXTERNAL_SURROUND_SOUND_KEY = "external_surround_sound_enabled";
 
+  private static boolean externalPcmOnlySound = false;
+  public static final int[] EXTERNAL_PCM_ONLY = {AudioFormat.ENCODING_PCM_16BIT,
+          AudioFormat.ENCODING_PCM_8BIT, AudioFormat.ENCODING_PCM_FLOAT, AudioFormat.ENCODING_MP3};
```

Comment:
I think `DEFAULT_AUDIO_CAPABILITIES` should be used instead.

Reply:
I think that should work Kevin.  Most everything supports 16 bit PCM ("WAV format").